csrc/id_model/id_model.cpp

```diff
@@ -778,6 +778,40 @@ StatefulInliningInfo buildStatefulInliningInfo(
       }
     }
 
+    if (auto mma_op = dynamic_cast<MmaOp*>(expr)) {
+      if (mma_op->isHopper()) {
+        auto producer_tvs = ir_utils::filterByType<TensorView>(expr->inputs());
+
+        auto producer_tv = producer_tvs.vector().front();
+
+        int64_t stage_slice_position = 4;
```

Comment:
How are you planning to remove this hard-coded value?

Reply:
Yes. This would refer to the `stage_slice_position` in `struct WarpSpecialized`.